requirements.txt

```diff
@@ -23,14 +23,5 @@ simplejson==3.16.0
 six==1.11.0
 uritemplate==3.0.0
 urllib3==1.24.1
-Django==2.1
```

Comment:
Why did you remove these packages? I'm sure alot of functionality depends on them

Reply:
it was duplicate